ibis/backends/mssql/tests/conftest.py

```diff
@@ -35,13 +35,23 @@ class TestConf(ServiceBackendTest):
     def test_files(self) -> Iterable[Path]:
         return self.data_dir.joinpath("csv").glob("*.csv")
 
+    def postload(self, **kw: Any):
+        self.connection = self.connect(database=IBIS_TEST_MSSQL_DB, **kw)
+
+    def _load_data(self, *, database: str = IBIS_TEST_MSSQL_DB, **_):
+        with self.connection._safe_raw_sql(
+            "IF DB_ID('ibis_testing') is NULL BEGIN CREATE DATABASE [ibis_testing] END"
+        ):
+            pass
```

Comment:
Nice!  I was getting cryptic user errors when I tried this, but I must've made a syntax error somewhere

Reply:
Probably was the other thing I had to change, `database=None` doesn't do what you might expect.